src/Console/Command/Validate.php

```diff
@@ -53,21 +58,55 @@ protected function configure(): void
             InputArgument::OPTIONAL,
             'The configuration file. (default: box.json, box.json.dist)'
         );
+        $this->addOption(
+            self::IGNORE_MESSAGES_OPTION,
+            'i',
+            InputOption::VALUE_NONE,
+            'Will not return a faulty code when a recommendation or warning is found'
+        );
     }
 
     /**
      * {@inheritdoc}
      */
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
+        $io = new SymfonyStyle($input, $output);
+
+        OutputConfigurator::configure($output);
+
         try {
-            $this->getConfig($input, $output);
+            $config = $this->getConfig($input, $output);
+
+            $recommendations = $config->getRecommendations();
+            $warnings = $config->getWarnings();
+
+            if ([] === $recommendations) {
```

Comment:
This block seems to be duplicated from above in `renderMessages` method - could it be extracted to a static helper perhaps?

Reply:
please don't create static code during fixing DRY rule violations, static code is hard to mock

Reply:
I extracted it in a dedicated class. It doesn't change how hard it is to mock it but I don't mock the commands since the integration tests are cheap enough and more reliable in our current case